src/bidders/prebid/prebid-helper.js

```diff
@@ -57,6 +57,19 @@ export function getBidByAdId(adId) {
 	return bids.length ? bids[0] : null;
 }
 
+export function getBidByAdUnitCode(adUnitCode, rendered = false) {
+	let bids = [];
+
+	if (window.pbjs && typeof window.pbjs.getBidResponsesForAdUnitCode === 'function') {
+		bids = window.pbjs.getBidResponsesForAdUnitCode(adUnitCode).bids || [];
+		bids = bids.filter(
+			bid => (rendered ? bid.status === 'rendered' : bid.status !== 'rendered')
```

Comment:
What about removing `rendered` flag and simply:
```
bid => bid.status !== 'rendered'
```

Reply:
Agree, changed.